resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -975,4 +984,45 @@ public class ApiClient implements AutoCloseable {
         @Override
         public void close() throws Exception { }
     }
+
+    private ApiClientConnectorResponse retryInvoke(ApiClientConnectorRequest connectorRequest, ApiClientConnectorResponse connectorResponse) {
+        int retries = 0;
+        long waitTimeInSec = 1L;
+        boolean retry;
+        int statusCode;
+        int MAX_RETRIES = 0;
+        long MAX_WAIT_INTERVAL_IN_SEC = 32L;
```

Comment:
Please suggest what should be the appropriate value for MAX_WAIT_INTERVAL_IN_SEC ?

Reply:
If it's a 429, use w/e the value is. If it's a 5xx, how about 60s?